mlir/lib/Dialect/MIOpen/Transforms/AffixTuningParameters.cpp

```diff
@@ -268,6 +268,7 @@ void AffixTuningParameters::affixTuningParametersImpl(T &op) {
     }
 
     op->setAttr("kpack", b.getI32IntegerAttr(validParams.gemmKPack));
+    op->setAttr("kblocks", b.getI32IntegerAttr(validParams.gemmKBlocks));
```

Comment:
Add a condition such that we set this attribute only when op is of `Conv2DBwdWeightOp`.

Reply:
will do.